webapp/package.json

```diff
@@ -12,7 +12,7 @@
 		"check": "eslint --ext .tsx,.ts . --quiet --cache && stylelint **/*.scss",
 		"fix": "eslint --ext .tsx,.ts . --quiet --fix --cache && stylelint --fix **/*.scss",
 		"fix:scss": "prettier --write './src/**/*.scss'",
-		"i18n-extract": "formatjs extract '../mattermost-plugin/webapp/src/**/*.{ts,tsx}' 'src/**/*.{ts,tsx}' --ignore '**/*.d.ts' '../**/*.d.ts' --out-file i18n/tmp.json && formatjs compile i18n/tmp.json --out-file i18n/en.json && rm i18n/tmp.json",
+		"i18n-extract": "formatjs extract \"../mattermost-plugin/webapp/src/**/*.{ts,tsx}\" \"src/**/*.{ts,tsx}\" --ignore \"**/*.d.ts\" \"../**/*.d.ts\" --out-file i18n/tmp.json && formatjs compile i18n/tmp.json --out-file i18n/en.json && npx rimraf i18n/tmp.json",
```

Comment:
@jufab what's the reasoning for changing `'` to an escaped `\"`? I don't believe that's something I've seen before in an npm script and it's not immediately obvious to me—is it specifically for Windows?

Reply:
did more digging, and yes, it's related to `cmd.exe` on Windows: https://github.com/eslint/eslint/issues/5796#issuecomment-206717297